react/components/PaymentMethodIcon.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import style from './PaymentMethodIcon.css'
+import withImage from './withImage'
+
+/**
+ * Shows an image for the payments forms accepted
+ */
+const PaymentMethodIcon: StorefrontFunctionComponent<
+  PaymentMethodIconProps
+> = ({ imageSrc, paymentMethod }) => {
+  if (!imageSrc) {
+    return null
+  }
+
+  return (
+    <img
+      className={`${style.paymentMethodIcon} w2 h2 ph2`}
```

Comment:
set this sizing on a parent container of img, and set only the images width to 100%